experiments/calibration/shared/observation_data_locations.jl

```diff
@@ -0,0 +1,41 @@
+era5_ds = Dataset(
+    joinpath(
+        ClimaLand.Artifacts.era5_surface_data2008_path(),
+        "era5_monthly_surface_fluxes_200801-200812.nc",
+    ),
+)
+
+# Make the ERA5 target
+ERA5_target = []
```

Comment:
You could use ClimaAnalysis to increase the level of abstraction for these operations (instead of dealing with the NCDataset directly)